NorthstarDLL/serverauthentication.h

```diff
@@ -49,7 +51,7 @@ class ServerAuthenticationManager
 	void RemovePlayer(R2::CBaseClient* player);
 	bool CheckDuplicateAccounts(R2::CBaseClient* player);
 	bool AuthenticatePlayer(R2::CBaseClient* player, uint64_t uid, char* authToken);
-	void VerifyPlayerName(R2::CBaseClient* player, char* authToken, char* name);
+	bool VerifyPlayerName(const char* authToken, const char* name);
```

Comment:
Value should be constant hence `const` and it allows for interaction with spdlog (`char *` sometimes give errors with spdlog but `const char *` works according to emma and blue).

Reply:
Yes?